src/qmflows/packages/cp2k_package.py

```diff
@@ -187,6 +187,18 @@ def __init__(self, settings: Optional[Settings],
                          work_dir=work_dir, properties=package_properties['cp2k'],
                          status=status, warnings=warnings)
 
+    @property
```

Comment:
Might it be an idea to add this to the ``Result`` superclass instead?

Reply:
done!